package.json

```diff
@@ -1,31 +1,66 @@
 {
   "name": "dx",
-  "version": "0.0.1",
+  "version": "0.0.3",
   "description": "Dx ordering workflow",
-  "main": "server.js",
+  "main": "./server/server.js",
   "scripts": {
-    "start": "node server.js"
+    "start": "node ./server/server.js",
+    "webpack": "webpack-dev-server --hot",
+    "build-css": "node-sass --include-path scss ./src/css/styles.scss   ./dist/css/styles.css",
+    "watch-css": "nodemon -e scss -x \"npm run build-css\""
   },
   "author": "Pranjal Kumar",
   "license": "MIT",
   "dependencies": {
-    "body-parser": "^1.15.2",
-    "cookie-parser": "^1.4.3",
-    "crypto-js": "^3.1.6",
     "ejs": "^2.5.1",
     "eslint": "^3.3.1",
     "express": "^4.14.0",
-    "mysql": "^2.11.1",
     "react": "^15.3.0",
     "react-dom": "^15.3.0",
+    "react-logger": "^1.1.0",
+    "react-redux": "^4.4.5",
+    "react-router": "^2.7.0",
+    "react-superagent": "^1.1.1",
+    "react-thunk": "^1.0.0",
+    "redux": "^3.5.2",
+    "redux-logger": "^2.6.1",
+    "redux-promise-middleware": "^4.0.0",
+    "redux-thunk": "^2.1.0",
+    "sequelize": "^3.24.1",
+    "superagent": "^2.2.0",
     "webpack": "^1.13.2",
     "webpack-dev-server": "^1.14.1"
   },
   "devDependencies": {
+    "babel-eslint": "^6.1.2",
+    "babel-plugin-transform-decorators-legacy": "^1.3.4",
+    "babel-preset-stage-2": "^6.13.0",
+    "babelify": "^7.3.0",
+    "browserify": "^13.1.0",
```

Comment:
There is no need for `browserify` and seems like it is not used. Remove all such unused packages.